style.css

```diff
@@ -0,0 +1,91 @@
+body{
+    font-family:serif;
+    font-size:19px;
+}
+img{
+    width:100%;
+}
+div.headerbackgroundimage{
+    background: url(http://letsbonus5.statics.download/products/164000/164679/13584378987941-0-680x276.jpg);
```

Comment:
You're technically allowed to include images from other sites like this, but it's not necessarily a good idea, because it means this other website controls an image that shows up on your site. Instead, download that image and save it in your site (I'd make an "images" folder).

Note that the image may be copyrighted which means you're not actually allowed to use it. Since this is a personal project and not a real commercial website, it's unlikely that anyone cares, but if you want to be safe, you can download images from sites that explicitly give you permission, like https://unsplash.com/